### PR TITLE
Force only supported commands on consumer table

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

File: common/consumer_table_pops.lua [hunk 1]
```diff
@@ -46,7 +46,14 @@ for i = n, 1, -3 do
...
        end
 
-   elseif op ~= 'flush' and op ~= 'flushresponse' and op ~= 'get' and op ~= 'getresponse' and op ~= 'notify' then
+   elseif
+       op == 'set' or
+       op == 'SET' or
+       op == 'create' or
+       op == 'remove' or
+       op == 'DEL' then
+   -- put entries into REDIS hash only when operations aare this types
```
Comment (lguohan) [inline common/consumer_table_pops.lua]: aare -> are

Comment (kcudnik) [reply common/consumer_table_pops.lua]: fixed

File: common/consumer_table_pops.lua [hunk 2]
```diff
@@ -46,7 +46,14 @@ for i = n, 1, -3 do
            st = st + 2
        end
 
-   elseif op ~= 'flush' and op ~= 'flushresponse' and op ~= 'get' and op ~= 'getresponse' and op ~= 'notify' then
+   elseif
+       op == 'set' or
+       op == 'SET' or
```
Comment (qiluo-msft) [inline common/consumer_table_pops.lua]: Why SET/DEL are upcases? Should we make them case insensitive?

Comment (kcudnik) [reply common/consumer_table_pops.lua]: thats a good question, maybe reducing to just lower case everything is good idea

**Pull request merged**